PlaygroundComponent.jsx

```diff
@@ -210,6 +220,8 @@ export const PlaygroundComponent = () => {
         <SelectRequestInput
           selected_value={text_data.selected_value}
           handleChange={handleSelectChange}
+          selected= {selected}
+          setSelected ={setSelected}
```

Comment:
unnecessary spaces.. I suggest to setup prettier and ESLint which there is already a card for it to avoid issues like this.